src/pyskindose/table_data/plot_HVL_data.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from pathlib import Path
+import os
+import plotly.graph_objects as go
+
+
+data_path = Path(os.path.abspath(__file__)).parent
+file_name = "HVL_simulated.csv"
```

Comment:
Do we need to set this variable as it is only used on row 10?

Reply:
nope, fixed

Reply:
plot script removed, we dont need it